### PR TITLE
Implement `Features::RG11B10UFLOAT_RENDERABLE`

### LINKED ISSUE #3268: Implement `Features::RG11B10UFLOAT_RENDERABLE`
https://github.com/gpuweb/gpuweb/pull/3170

### PR DESCRIPTION
**Pull request by jinleili**
**Checklist**

- [x] Run `cargo clippy`.
- [x] Run `RUSTFLAGS=--cfg=web_sys_unstable_apis cargo clippy --target wasm32-unknown-unknown` if applicable.

**Connections**
https://github.com/gfx-rs/wgpu/pull/3689
resolves https://github.com/gfx-rs/wgpu/issues/3268

**Testing**
Tested locally

### CODE REVIEW

**Review by crowlKats — COMMENTED** [review]
Deno part LGTM

File: wgpu/src/backend/web.rs [hunk 1]
```diff
@@ -1099,7 +1099,7 @@ impl crate::context::Context for Context {
         _adapter_data: &Self::AdapterData,
         format: wgt::TextureFormat,
     ) -> wgt::TextureFormatFeatures {
-        format.guaranteed_format_features()
+        format.guaranteed_format_features(wgt::Features::empty())
```
Comment (teoxoy) [inline wgpu/src/backend/web.rs]: I think we should pass the actual adapter features here or else validation will fail even if the feature is actually supported.

**Review by teoxoy — CHANGES_REQUESTED** [review]
Besides the comment it looks great! Thanks!

**Review by cwfitzgerald — COMMENTED** [review]
After comment is addressed secondary g2g :)

**Review by teoxoy — APPROVED** [review]
Good stuff!

**Comment by teoxoy** [comment]
@jinleili would you like to also reference this PR and your username in the existing changelog entry?

**Comment by jinleili** [comment]
> @jinleili would you like to also reference this PR and your username in the existing changelog entry?

I prefer not to reference my username in the changelog. Thanks.

**Pull request merged**